lib/rules/quotes.js

```diff
@@ -19,7 +19,7 @@ const QUOTE_SETTINGS = {
     double: {
         quote: "\"",
         alternateQuote: "'",
-        description: "doublequote"
+        description: "Doublequote"
```

Comment:
It seems that this was mistakenly changed and that's why the tests are failing.

Reply:
done